### DISCUSSION TITLE
flash espixel on wt32-eth

### DISCUSSION THREAD

**Discussion by Francesco-Rugg** [forkineye/ESPixelStick#591] [Q&A]
I read about this board and ethernet support on https://github.com/forkineye/ESPixelStick/issues/268#top
https://github.com/forkineye/ESPixelStick/issues/268 to test this board and the ethernet support what I need to do?
I'm a newbie and I have 3 wt32 with wled.
I don't need the sd card but only an ethernet pixel node and swap from wled to espixel that I already run on 10 wemos d1 mini in my light show.
I only need to download the [ESPixelStick v4.0-beta4](https://github.com/forkineye/ESPixelStick/releases/tag/v4.0-beta4) and flash to the wt32 with the esp tool flasher?

**Comment by MartinMueller2003** [comment]
No. That image is out of date and does not support Wired Ethernet. Use this image:
https://drive.google.com/file/d/1DG3wz8lbYNc371OIbeOqkzMaY5Zd2SKA/view?usp=sharing

**Reply by Francesco-Rugg** [reply]
ok thank you, but where I can find this file from google drive here on github?
or is a your release

**Comment by MartinMueller2003** [comment]
I keep the latest images in a dist.zip file at that link. Click the link to download the file. Unzip it on your machine and run it.

**Comment by MartinMueller2003** [comment]
Latest is now on https://espixelstickwebflasher.from-ct.com:5000/
